app/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -229,89 +249,91 @@ export class SimulatedMockLink extends ApolloLink {
   }
 }
 
-export const mockExperimentQuery = (
+export function mockExperimentQuery<
+  T extends getExperiment["experimentBySlug"] = getExperiment_experimentBySlug
+>(
```

Comment:
"`T` can be `getExperiment_experimentBySlug` or `null`, defaulting to `getExperiment_experimentBySlug`"